packages/core-mobile/src/hooks/useHomeScreenHeader.tsx

```diff
@@ -0,0 +1,37 @@
+// import { SignedInStackProps } from 'navigation-new/types'
+import React, { useCallback, useLayoutEffect } from 'react'
+import { View } from 'react-native'
+import { Link, useNavigation } from 'expo-router'
+
+const useHomeScreenHeader = (): void => {
+  const navigation = useNavigation()
+  const renderHeaderLeft = useCallback(
+    () => (
+      <View style={{ marginLeft: 12 }}>
+        <Link href="/settings/">Account</Link>
+      </View>
+    ),
+    []
+  )
+
+  const renderHeaderRight = useCallback(
+    () => (
+      <View style={{ flexDirection: 'row', gap: 12, marginRight: 12 }}>
+        <Link href="/receive/">Receive</Link>
+        <Link href="/notifications/">Notifications</Link>
+      </View>
+    ),
+    []
+  )
+
+  useLayoutEffect(() => {
```

Comment:
is it possible to statically configure these in advanced? that will be preferred as using hook can affect performance

Reply:
I'm not sure how we can achieve this at the moment. I don't want to replace the entire navigation bar(like we're currently doing with `<TopNavigationHeader />`) because I want to keep their animations and behavior with left/right/back bar buttons and title view. We can revisit this in another pr once we come up with a better solution.

Reply:
configured statistically. :D